deploy/Chart/templates/controller/webhook.yaml

```diff
@@ -32,7 +33,7 @@ webhooks:
 - admissionReviewVersions:
   - v1
   clientConfig:
-    caBundle: {{ b64enc $ca.Cert }}
+    caBundle: {{ $validatingWebhookCaBundle }}
```

Comment:
@ytimocin Can you please make the code consistent with how UCP does? UCP way is more intuitive.

Reply:
In UCP's case, we don't need to select a **caBundle** from an array of webhook configurations: https://github.com/radius-project/radius/blob/main/deploy/Chart/templates/ucp/apiservice.yaml#L42.

The reason I did it this way is that we have a new webhook coming soon and that will also be added to the array of configs. We will need to map the right webhook configuration to the right caBundle.

The only thing that we will need to do for the next webhook is to call the function with the right webhook name.

Let me know if that makes sense @youngbupark.

Reply:
For mutatingAdmissionController, we will use existing `controller` service and share the same endpoint. Then I do not think we need additional cert for mutating controller.